crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -62,7 +62,7 @@ impl headers::Header for ShowUsage {
     }
 
     fn encode<E: Extend<HeaderValue>>(&self, _: &mut E) {
-        unimplemented!()
+        // unimplemented!()
```

Comment:
I think we want this one

Reply:
/// Encode this type to a `HeaderMap`.
    ///
    /// This function should be infallible. Any errors converting to a
    /// `HeaderValue` should have been caught when parsing or constructing
    /// this value.
    fn encode<E: Extend<HeaderValue>>(&self, values: &mut E);

Reply:
We do want this one, because we should never be encoding this header.

Reply:
ok